UI/MainScreen.cpp

```diff
@@ -1389,7 +1389,7 @@ void MainScreen::update() {
 	UpdateUIState(UISTATE_MENU);
 
 	if (searchChanged_) {
-		for (auto browser : gameBrowsers_)
+		for (auto* browser : gameBrowsers_)
```

Comment:
just wondering, is this made a difference? since `gameBrowsers_`'s elements is already a pointer (ie. `GameBrowser*`), thus `browser` type will became `GameBrowser**` after this change isn't?

Reply:
fix it here https://github.com/hrydgard/ppsspp/pull/18959/commits/83cb897b1fd92965583adc608edf4b246f3c41b1

Reply:
> just wondering, is this made a difference? since `gameBrowsers_`'s elements are already a pointer (ie. `GameBrowser*`), thus `browser` type will became `GameBrowser**` after this change isn't?

Auto is like a template, so no.
* `auto browser` => `auto` turns into `GameBrowser*`.
* `auto* browser` => `auto` turns into `GameBrowser`.
The type of `browser` is still `GameBrowser*`.